rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildExecutionConfigurationRest.java

```diff
@@ -95,6 +95,18 @@ public BuildExecutionConfiguration toBuildExecutionConfiguration() {
         );
     }
 
+    public BuildConfigurationAuditedRest createBuildConfigurationAuditedRest() {
```

Comment:
@matejonnet see https://github.com/project-ncl/pnc/pull/878#discussion_r55341555. Now the relation is between 2 rest-model objects, not anymore between a rest-model object and a db entity. I could not avoid this conversion (from BuildExecutionConfigurationRest to BuildConfigurationAuditedRest), as the rsql relies on the presence of this object also for the running builds, for being able to apply the filters.